### DISCUSSION TITLE
Dismissing indexing status window

### DISCUSSION THREAD

**Discussion by xim** [neoclide/coc.nvim#4050] [Q&A] [answered]
I use ccls to index a huge project. This works quite well, but occasionally the status window gets stuck at 99% indexing:

![image](https://user-images.githubusercontent.com/169057/184363492-fc998441-6e51-48b9-b644-43faf6c60b49.png)

At the time the computer / ccls are idle.

This is a bit distracting and I would like to be able to dismiss it in general. Is there a way to manually dismiss it?

**Comment by chemzqm** [comment, marked as answer]
`:h coc-config-notification-disabledProgressSources`

**Comment by xim** [comment]
If anyone else wants to do this runtime:

`:call coc#config('coc.preferences', {'notification.disabledProgressSources': ['*']})`

Note that it appears to take a (little) while before it is heeded.

**Reply by xim** [reply]
I wrote too soon ... This state doesn't hit me consistently, and when I tested it twice the progress indicator actually disappeared accidentally. After trying again a third time, it actually stayed up forever. Is there something wrong with how I am calling this, or can't this be applied like this at runtime?

**Reply by chemzqm** [reply]
You should use 
```vim
:call coc#config('notification', {'disabledProgressSources': ['*']})
```
instead

**Reply by chemzqm** [reply]
Use coc-settings.json is recommended.

**Reply by xim** [reply]
I wasn't able to make that work, but this alternative approach works for me and I couldn't see any immediate drawbacks:

```vim
nnoremap <silent> gx :call coc#notify#close_all()<cr>
```
